src/styles/partials/components/_icon-button.scss

```diff
@@ -29,3 +29,65 @@
 		background-color: $vibrant-blue;
 	}
 }
+.dg_icon-buttonDefault {
+	color: $black;
+	display: block;
+	font-family: $default-heading-font;
+	font-weight: $font-bold;
+	text-align: center;
+	text-decoration: none;
+	text-transform: uppercase;
+	border: 2px solid $black;
+	padding: 50px 30px 50px 30px;
+	transition: all 0.3s ease 0s;
+	i,
+	svg {
```

Comment:
not sure where the svg is coming from here, we aren't using svg's that i know of.